pkg/capacityscheduling/elasticquota.go

```diff
@@ -52,20 +52,22 @@ func (e ElasticQuotaInfos) aggregatedMinOverUsedWithPod(podRequest framework.Res
 // ElasticQuotaInfo is a wrapper to a ElasticQuota with information.
 // Each namespace can only have one ElasticQuota.
 type ElasticQuotaInfo struct {
-	Namespace string
-	pods      sets.String
-	Min       *framework.Resource
-	Max       *framework.Resource
-	Used      *framework.Resource
+	Namespace    string
+	pods         sets.String
+	deletingPods map[string]*v1.Pod
```

Comment:
Not quite a fan of using this variable. Could you explain the reasons? Why not just update the cache upon receiving Pod Deleting events.

Reply:
I delete the variable to store the deleting pods.